source/2020-09-11-the-ember-times-issue-163.md

```diff
@@ -0,0 +1,109 @@
+---
+title: The Ember Times - Issue No. 163
+author: Isaac Lee, Jared Galanis, Abhilash LR, Amy Lam
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2020
+alias : "blog/2020/09/11-the-ember-times-issue-163.html"
+responsive: true
+---
+
+👋 Emberistas! 🐹
+
+Learn what's new in the Ember 3.21 release 🎉,
+Embroider preparation blog post 📖,
+get ready for Hacktoberfest 2020 👩🏽‍💻🧑🏽‍💻,
```

Comment:
```suggestion
Get ready for Hacktoberfest 2020 👩🏽‍💻🧑🏽‍💻,
```

Reply:
thanks, let's keep it as-is, is kinda one long sentence 

![image](https://user-images.githubusercontent.com/1372946/92985729-23a5ca80-f46a-11ea-8732-b02b3c2644a8.png)

Reply:
Ah! I didn't realize this. Sounds good @amyrlam